homepage/src/pages/resource.jsx

```diff
@@ -0,0 +1,97 @@
+import Banner from '../components/banner';
+import TwoColumns from '../components/twoColumns';
+import ThreeColumns from '../components/threeColumns';
+import ImageAndText from '../components/imageAndText';
+
+// ? Not used in index.md, just for Generic part
+import Section from '../components/section';
+import Headline from '../components/headline';
+
+import { fetchPage } from '../lib/fetchPage';
+
+/**
+ *
+ * @type {import('next').GetStaticProps}
+ */
+export const getStaticProps = async ({ locale }) => {
+  const page = fetchPage(locale, 'index');
+
+  return {
+    props: {
+      page,
+    },
+  };
+};
+
+export default function Resource({ page }) {
+  return (
+    <>
+      {page.data['layout_list'].map((layout) => {
```

Comment:
`page.data.layout_list` would be undefined for some case. We shall change to `page.data['layout_list']?.map`.

Reply:
fixed.

Reply:
fixed.